### DISCUSSION TITLE
Ability to delete notifications

### DISCUSSION THREAD

**Discussion by BillXWB** [community/community#13482] [Other Feature Feedback, Questions, & Ideas]
When I try to unsubscribe a notification in the inbox, it is just marked as `Done`. This is a little bit weird. I want to be able to truly remove a notification from the list.

**Comment by queenofcorgis** [comment]
Thanks for your feedback @BillXWB! What would this look like ideally for you?

**Reply by BillXWB** [reply]
Let the notification disappear when you click the `Unsubscribe` button in the inbox. Right now, it will be marked as `Done` and you still can see it in the `Done` list.

**Reply by BillXWB** [reply]
Or can we add a `Delete` button to do this?

**Reply by queenofcorgis** [reply]
Thank you for sharing, this context will help our team as they review feedback 😄

**Reply by rushi47** [reply]
Delete option is much needed I second this, cause it's all cluttered now.

**Reply by radar-blip** [reply]
3rd
